app/views/locations/countries/show.html.erb

```diff
@@ -32,6 +32,10 @@
               <td><%= t('activerecord.attributes.country.mernis_code') %></td>
               <td><%= @country.mernis_code %></td>
             </tr>
+            <tr>
+              <td><%= t('activerecord.attributes.country.yoksis_code') %></td>
```

Comment:
Buradaki başlık çevirilerini sadece kolon ismi ile yapabilirmisin. Çeviriler show action'ı altında da bulunduğu için uzun bir şekilde çevirinin konumunu belirtmene gerek yok. Diğer sayfalarda değiştirmiştik burada kalmış, sen hepsini aşağıdaki örnekteki yaparsın
Örnek: `t(.yoksis_code)`

Reply:
Hallettim :+1: a7f67a3ca686b4aadf6033359cbe8a6ca62548cd